service/src/main/java/bio/terra/landingzone/stairway/flight/CromwellStepsDefinitionProvider.java

```diff
@@ -0,0 +1,104 @@
+package bio.terra.landingzone.stairway.flight;
+
+import bio.terra.landingzone.common.utils.RetryRules;
+import bio.terra.landingzone.library.landingzones.definition.ArmManagers;
+import bio.terra.landingzone.library.landingzones.definition.ResourceNameGenerator;
+import bio.terra.landingzone.library.landingzones.definition.factories.ParametersResolver;
+import bio.terra.landingzone.stairway.flight.create.resource.step.AggregateLandingZoneResourcesStep;
+import bio.terra.landingzone.stairway.flight.create.resource.step.CreateAksStep;
+import bio.terra.landingzone.stairway.flight.create.resource.step.CreateAppInsightsStep;
+import bio.terra.landingzone.stairway.flight.create.resource.step.CreateBatchAccountStep;
+import bio.terra.landingzone.stairway.flight.create.resource.step.CreateBatchLogSettingsStep;
+import bio.terra.landingzone.stairway.flight.create.resource.step.CreateLogAnalyticsDataCollectionRulesStep;
+import bio.terra.landingzone.stairway.flight.create.resource.step.CreateLogAnalyticsWorkspaceStep;
+import bio.terra.landingzone.stairway.flight.create.resource.step.CreatePostgresLogSettingsStep;
+import bio.terra.landingzone.stairway.flight.create.resource.step.CreatePostgresqlDbStep;
+import bio.terra.landingzone.stairway.flight.create.resource.step.CreatePrivateEndpointStep;
+import bio.terra.landingzone.stairway.flight.create.resource.step.CreateRelayNamespaceStep;
+import bio.terra.landingzone.stairway.flight.create.resource.step.CreateStorageAccountCorsRules;
+import bio.terra.landingzone.stairway.flight.create.resource.step.CreateStorageAccountStep;
+import bio.terra.landingzone.stairway.flight.create.resource.step.CreateStorageAuditLogSettingsStep;
+import bio.terra.landingzone.stairway.flight.create.resource.step.CreateVnetStep;
+import bio.terra.landingzone.stairway.flight.create.resource.step.GetManagedResourceGroupInfo;
+import bio.terra.stairway.RetryRule;
+import bio.terra.stairway.Step;
+import java.util.List;
+import org.apache.commons.lang3.tuple.Pair;
+
+public class CromwellStepsDefinitionProvider implements StepsDefinitionProvider {
+  // TODO: this doesn't take into account versioning
```

Comment:
Do you have ideas about how versioning support can be added later?

Reply:
Yes. I think it might be incorporated into each steps definition provider like (CromwellStepsDefinitionProvider). Instead of list there will be a map: v1->definition (list of steps)